apps/storybook-react/stories/Icon.stories.tsx

```diff
@@ -54,7 +54,25 @@ export default {
   component: Icon,
   argTypes: {
     name: {
+      control: {
+        type: 'select',
+        options: [
```

Comment:
Suggest changing options to use `Object.keys(icons)`, so it will be updated dynamically if we change icons used in example.